Fix #48

melkor54248 opened the pull request.

NotMyself commented:
Hi, @melkor54248 thanks for taking a stab at this. It looks like your PR doesn't really change anything. From what I can tell, you implemented a string interpolation step and discovered someone else already did it?

I am going to close this PR and look at your other one.

melkor54248 commented:
Yes that's right so I reverted tese commits but in the end I fixed the steps  numbering because step 22 was duplicate. That's commit 3977f63. Maybe you could merge this?

NotMyself commented:
Sure thanks.
